rules/config

```diff
@@ -44,6 +44,9 @@ DEFAULT_PASSWORD = YourPaSsWoRd
 # If not set (default behavior) the default minigraph built into the image will be used.
 # ENABLE_DHCP_GRAPH_SERVICE = y
 
+# ENABLE_PDE - Enable platform development enviroment
```

Comment:
Update description to clarify that option determines whether we will build the PDE container into the SONiC image.

Also typo: s/enviroment/environment/

Reply:
Typo still exists: s/enviroment/environment/

Reply:
Fixed spelling error.